elastic/kibana
# [Discover] Fix visibility of saved search grids when one of them is in fullscreen mode

Closes https://github.com/elastic/kibana/issues/134032

## Summary

This PR adds CSS which overrides Dashboard panel styles when one of grids is in fullscreen mode.

Steps to test:
1. Create a dashboard
2. Add multiple Saved Searches to it (next to each other)
3. Try fullscreen mode for any of them => only the expanded grid should be visible in its fullscreen mode

### Checklist
